policy/modules/xen/ndvm.te

```diff
@@ -50,6 +51,7 @@ stubdom_event_src(ndvm_t, stubdom-ndvm_evchn_t)
 # Local Policy
 #
 
+dom0_ioemu(ndvm_t)
```

Comment:
What was the motivation for adding this rule?

Reply:
I fixed up svirt-interpose and qemu running in dom0, and they need this.

Reply:
Gotcha.  Thanks.